### PR TITLE
Fix broken docs links in ecommerce docs

### PR DESCRIPTION
**Pull request by schenedx**
@mjfrey @clintonb Please review
Just trying to clean up the errors when compiling docs on ecommerce

### CODE REVIEW

**Review by mjfrey — APPROVED** [review]

File: docs/index.rst [hunk 1]
```diff
@@ -23,6 +21,7 @@ use the E-Commerce service with the Open edX platform.
    test_features
    test_ecommerce
    additional_features/index
+   theming
```
Comment (bderusha) [inline docs/index.rst]: `theming` already exists on line 17

File: docs/create_products/create_products_overview.rst [hunk 2]
```diff
@@ -4,8 +4,6 @@
 Creating Products Overview
 ###########################
 
-.. include:: ../products_desc.txt
```
Comment (bderusha) [inline docs/create_products/create_products_overview.rst:7]: Do we know why this is broken?  Isn't this information still relevant to include here?

File: docs/index.rst [hunk 3]
```diff
@@ -13,8 +13,6 @@ use the E-Commerce service with the Open edX platform.
 .. toctree::
    :maxdepth: 2
 
-   about_ecommerce
-   flow
```
Comment (bderusha) [inline docs/index.rst:17]: Same question as above:
 Do we know why this is broken?  Isn't this information still relevant to include here?

**Review by bderusha — CHANGES_REQUESTED** [review]

File: docs/index.rst (hunk 1 above)
Comment (schenedx) [reply docs/index.rst]: I'll remove that

File: docs/index.rst (hunk 3 above)
Comment (schenedx) [reply docs/index.rst:17]: @bderusha Looks like these broken links are added as part of https://github.com/edx/ecommerce/commit/e85bad23373c314c28b39ef91d7aa177d97af65e
The commit does not have any PR associated. I remember David had trouble revert and re-introduce commits when he is working on this task.
I think the thing to do that make most sense is to revert commit e85bad23373c314c28b39ef91d7aa177d97af65e

Comment (bderusha) [reply docs/index.rst:17]: Agreed.  Let's revert the commit and make sure `theming` gets added back if that is working.

**Review by bderusha — APPROVED** [review]
:shipit:

**Pull request merged**